src/libponyrt/actor/actor.c

```diff
@@ -589,3 +656,45 @@ PONY_API void pony_poll(pony_ctx_t* ctx)
   pony_assert(ctx->current != NULL);
   ponyint_actor_run(ctx, ctx->current, 1);
 }
+
+void ponyint_actor_setoverloaded(pony_actor_t* actor)
+{
+  set_flag(actor, FLAG_OVERLOADED);
+}
+
+bool ponyint_actor_overloaded(pony_actor_t* actor)
+{
+  return has_flag(actor, FLAG_OVERLOADED);
+}
+
+void ponyint_actor_unsetoverloaded(pony_ctx_t* ctx, pony_actor_t* actor)
+{
+  unset_flag(actor, FLAG_OVERLOADED);
+  if (!has_flag(actor, FLAG_UNDER_PRESSURE))
+    ponyint_sched_unmute_senders(ctx, actor, true);
+}
+
+PONY_API void pony_apply_backpressure()
```

Comment:
I think this should follow the convention for runtime functions and take a `pony_ctx_t*` parameter.

Reply:
This was intentionally done this way to make calling from Pony straightforward. Sylvan C and I spent a while coming up w this approach

Reply:
Ok, that makes sense.